tests/v2/validation/snapshot/snapshot.go

```diff
@@ -327,10 +314,10 @@ func snapshotV2Prov(t *testing.T, client *rancher.Client, podTemplate corev1.Pod
 		}
 	}
 
-	return initialSnapshotConfig{initialKubernetesVersion, initialControlPlaneConcurrencyValue, initialWorkerConcurrencyValue, snapshotToRestore}
+	return restoreSnapshotConfig, initialSnapshotConfig{initialKubernetesVersion, initialControlPlaneConcurrencyValue, initialWorkerConcurrencyValue, snapshotToRestore}
 }
 
-func restoreV2Prov(t *testing.T, client *rancher.Client, v2prov initialSnapshotConfig, etcdRestore *etcdsnapshot.Config, clusterName, clusterID string) {
+func restoreV2Prov(t *testing.T, client *rancher.Client, v2prov initialSnapshotConfig, etcdRestore *etcdsnapshot.Config, cluster *apisV1.Cluster, clusterName, clusterID string) {
```

Comment:
since you're loading the whole cluster object, we don't need clusterName / ClusterID . also see previous comments on cutting out the initialSnapshotConfig now that we're sending the whole cluster object.

Reply:
Resolved.